packages/commonwealth/client/scripts/views/components/component_kit/new_designs/CWSearchBar/CWSearchBar.tsx

```diff
@@ -64,6 +70,12 @@ export const CWSearchBar: FC<SearchBarProps> = ({
     'members',
   ]);
 
+  useEffect(() => {
+    if (size === 'small') {
+      inputRef?.current?.focus?.();
+    }
+  }, [size]);
+
```

Comment:
now sure why we need to do this, do we ever change `size` prop based on screen size?

Reply:
so the search component is complicated and it definitely needs refactor but first it needs rethink for desktop and mobile UX. Right know I used current searchbar component and just adjusted it to work on mobile. Yes, on mobile we use size=small and this useEffect makes input to focus when the modal appears

https://github.com/hicommonwealth/commonwealth/assets/14819225/07f246a9-7b2c-4a3d-9eb1-98b3b6ad998a

Reply:
That makes sense, thanks!